dali/kernels/slice/slice_kernel_utils.h

```diff
@@ -32,7 +32,10 @@ template <typename T, int Dims>
 struct SliceArgs {
   TensorShape<Dims> anchor;
   TensorShape<Dims> shape;
-  SmallVector<T, 8> fill_values = {0, };
+  TensorShape<Dims> step;
```

Comment:
The tests fail, because `step` is initialized to all zeros.
```suggestion
  TensorShape<Dims> step = UnitCubeShape<Dims>();
```
Currently there's no such function. You can add one or just fill `step` with ones in a constructor.